src/Microsoft.ML.Data/StaticPipe/StaticSchemaShape.cs

```diff
@@ -234,9 +235,8 @@ private static bool IsStandardCore(Type t)
         /// <param name="col">The column</param>
         /// <returns>The .NET type for the static pipelines that should be used to reflect this type, given
         /// both the characteristics of the <see cref="ColumnType"/> as well as one or two crucial pieces of metadata</returns>
-        private static Type GetTypeOrNull(IColumn col)
+        private static Type GetTypeOrNull(Schema.Column col)
         {
-            Contracts.AssertValue(col);
```

Comment:
> Contracts.AssertValue(col); [](start = 11, length = 28)

curious, why remove it?

Reply:
Hi @sfilipi thanks for looking. You'll note that I've shifted usage of `IColumn` to use `Schema.Column`, which is a struct, so checks against it being `null` are no longer sensible. Indeed if you try to keep this, you will see an error.